src/Browser.elm

```diff
@@ -134,7 +134,7 @@ app shows an accurate date in the title, etc.
 > not the place to manage CSS assets. If you want to work with CSS, there are
 > a couple ways:
 >
-> 1. Packages like [`rtfeldman/elm-css`][elm-css] give all of the features
+> 1. Packages like `https://github.com/rtfeldman/elm-css` give all of the features
```

Comment:
Revert this and change the link destination at line 153 to `/packages/rtfeldman/elm-css/latest/`
```suggestion
> 1. Packages like [`rtfeldman/elm-css`][elm-css] give all of the features
```

Reply:
Yep - right after posting I noticed line 153!

And you're right that the link to https://package.elm-lang.org/packages/rtfeldman/elm-css/latest/ is more correct than what I proposed.